_events/2018-06-27-de.STAIR-primer-rna-seq.md

```diff
@@ -0,0 +1,24 @@
+---
+site: Jena
```

Comment:
@bagnacan You need to change it to `main`. The site here means on which sub-website it appears: `main` for https://usegalaxy-eu.github.io/ or `freiburg` for https://usegalaxy-eu.github.io/freiburg/